apps/estuary/config/integration.exs

```diff
@@ -2,6 +2,8 @@ use Mix.Config
 
 config :estuary,
   divo: [
-    {DivoKafka, [create_topics: "topic1:1:1,topic2:1:1:compact", outside_host: "localhost"]}
+    {DivoKafka, [create_topics: "event-stream-integration:1:1", outside_host: "localhost"]}
```

Comment:
do we need to create a topic here anymore since our app does it?

Reply:
We will need this if we move the dev.exs to integration.exs, but not if we move it to config.exs